ts/Core/Tooltip.ts

```diff
@@ -449,7 +449,7 @@ class Tooltip {
                 css(container, {
                     position: 'absolute',
                     top: '1px',
-                    pointerEvents,
+                    pointerEvents: 'none',
```

Comment:
I would add a short description with the issue number to describe, why `'none'` is needed here.